app/components/goodcity/designate-form.js

```diff
@@ -15,6 +21,14 @@ export default Ember.Component.extend({
     );
   }),
 
+  onOrderChange: Ember.observer("order", function() {
+    this.set("quantity", this.get("maxQuantity"));
+  }),
+
+  onItemChange: Ember.observer("pkg", function() {
+    this.set("quantity", this.get("maxQuantity"));
+  }),
+
```

Comment:
```suggestion
  onRecordsChange: Ember.observer("order", "pkg", function() {
    this.set("quantity", this.get("maxQuantity"));
  }),
```

Reply:
Adding multiple properties in the observer does not support it. Verified above suggested change on local.